tests/test_language_server.py

```diff
@@ -48,6 +48,22 @@ def test_bf_initialize(client_server):
     assert hasattr(response, 'capabilities')
 
 
+def test_bf_initialize_spec(client_server):
+    client, _ = client_server
+    # Use a dictionary to send only the elements marked
+    # as required
+    response = client.lsp.send_request(
+        INITIALIZE,
+        {
```

Comment:
it might be clearer just to have lines 57-61 be an empty _dict_:
```
response = client.lsp.send_request(
    INITIALIZE,
    {}
).result(timeout=CALL_TIMEOUT)
```

Reply:
How do we test for paremeters required by the specification then?